homeworks/olena.ambitska_elena-ambitska/GamesProject/style.css

```diff
@@ -0,0 +1,65 @@
+
+body {
+  padding: 0 16px;
+  color: var(--black);
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+.section {
+  padding: 24px 0;
+}
+
+.article {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.section__image {
+  max-height: 356px;
+  width: 100%;
+  max-width: 536px;
+  padding-bottom: 24px;
+}
+
+.section__title {
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 1.2;
```

Comment:
The result `line-height` is different than on the design, but, proly, your way of setting it is more correct.

Reply:
So, there is no need to change your `1.2` value